portal/src/model/themeAuthFlowV2.ts

```diff
@@ -7,8 +7,42 @@ import {
   CssRuleNodeWrapper,
 } from "../util/cssVisitor";
 
+export enum Theme {
+  Light = "light",
+  Dark = "dark",
+}
+
 export const enum ThemeTargetSelector {
   Light = ":root",
+  Dark = ":root.dark",
+}
+export function getThemeTargetSelector(theme: Theme): ThemeTargetSelector {
+  switch (theme) {
+    case Theme.Light:
+      return ThemeTargetSelector.Light;
+    case Theme.Dark:
+      return ThemeTargetSelector.Dark;
+    default:
+      return ThemeTargetSelector.Light;
+  }
+}
+
+export function selectByTheme<T>(option: { [t in Theme]: T }, theme: Theme): T {
+  return option[theme];
+}
+
+export const enum CSSVariable {
+  LayoutFixAlignItems = "--layout-flex-align-items",
+  LayoutPaddingLeft = "--layout-padding-left",
+  LayoutPaddingRight = "--layout-padding-right",
+  LayoutBackgroundColor = "--layout__bg-color",
+  LayoutBackgroundImage = "--layout__bg-image",
+  PrimaryButtonBackgroundColor = "--primary-btn__bg-color",
+  PrimaryButtonTextColor = "--primary-btn__text-color",
+  PrimaryButtonBorderRadius = "--primary-btn__border-radius",
+  InputFiledBorderRadius = "--input__border-radius",
```

Comment:
The border radius is lost after changed to other option and back to middle. Same problem in input border radius.
https://github.com/authgear/authgear-server/assets/34942210/f3d329c3-cbac-4cfe-a3ee-ecf3d817618c